pvlib/tests/iotools/test_solargis.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+import pytest
+import pvlib
+import requests
+from ..conftest import (RERUNS, RERUNS_DELAY, assert_frame_equal,
+                        assert_index_equal)
+
+
+@pytest.fixture
+def hourly_index():
+    hourly_index = pd.date_range(start='2022-01-01 00:30+01:00', freq='60min',
+                                 periods=24, name='dateTime')
+    hourly_index.freq = None
+    return hourly_index
+
+
+@pytest.fixture
+def hourly_index_start_utc():
+    hourly_index_left_utc = pd.date_range(
+        start='2023-01-01 00:00+00:00', freq='30min', periods=24*2,
+        name='dateTime')
+    hourly_index_left_utc.freq = None
+    return hourly_index_left_utc
+
+
+@pytest.fixture
+def hourly_dataframe(hourly_index):
+    ghi = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 5.0, 73.0, 152.0, 141.0, 105.0,
+           62.0, 65.0, 62.0, 11.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+    dni = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 30.0, 233.0, 301.0, 136.0, 32.0,
+           0.0, 3.0, 77.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+    return pd.DataFrame(data={'ghi': ghi, 'dni': dni}, index=hourly_index)
+
+
+@pytest.mark.skipif(pd.__version__ < '1.3.0',
+                    reason='pd.read_xml is new as of 1.3.0')
```

Comment:
I don't think there's a good alternative way around it. But yes, seems silly to increase the version by this much because of one function. At the same time, just bumping up the required version is surely the best way to reduce the maintenance burden - so that's my vote.

Reply:
I vote bump version & not keep super old minimum

Reply:
Also bumping Numpy:

![image](https://github.com/pvlib/pvlib-python/assets/39184289/32389558-1ea8-4e1f-80ab-9bdaa6b10051)